subgraphs/burrow/src/handlers/actions.ts

```diff
@@ -236,8 +274,22 @@ export function handleWithdraw(event: EventData): void {
     .times(token.lastPriceUSD!);
 
   // usage metrics
-  if (account._last_active_timestamp.lt(usageDailySnapshot.timestamp)) {
+  const hourlyActiveAccount = ActiveAccount.load(
+    "HOURLY-"
```

Comment:
Use a constant. We try to use constants for any string literal or number. You don't need to worry about "-" though